config/travis/runtests.sh

```diff
@@ -62,16 +58,7 @@ then
 
 elif test "${TARGET}" = "dockerfile";
 then
-	SOURCE_PATH=${PWD};
-	CONTAINER_NAME="log2timeline/plaso";
-
-	cd config/docker
-
-	docker build --build-arg PPA_TRACK="dev" -f Dockerfile -t ${CONTAINER_NAME} .
-
-	docker run -v ${SOURCE_PATH}:/data ${CONTAINER_NAME} log2timeline --status_view linear /data/test.plaso /data/test_data;
-
-	docker run -v ${SOURCE_PATH}:/data ${CONTAINER_NAME} psort --status_view linear -w /data/timeline.log /data/test.plaso;
```

Comment:
Does that mean we don't run l2t & plaso from the container anymore?

Reply:
Good catch. These changes need to be undone and l2tdevtools needs some updating to generate these.